### DISCUSSION TITLE
How to use `text` with `geo` , show name on each state?

### DISCUSSION THREAD

**Discussion by winner106** [observablehq/plot#1650] [Q&A] [answered]
I  want to show the name of state on the map, how to implement? Thanks...

[https://observablehq.com/@observablehq/plot-albers-usa-projection](https://observablehq.com/@observablehq/plot-albers-usa-projection)

**Comment by mbostock** [comment, marked as answer]
Use the [centroid transform](https://observablehq.com/plot/transforms/centroid).

Here is an example using the tip mark: https://observablehq.com/plot/marks/tip

**Reply by winner106** [reply]
Thank you very much for your suggestion. I highly appreciate it. I would like to kindly suggest adding a link to the `Centroid` in the documentation of the geo mark to indicate the possibility of adding labels. 

The documentation is quite extensive, and it can be challenging to read it all at once. Thank you again for your attention to this matter.
